### DISCUSSION TITLE
Which QEMU version in future? Whats about 32 Bit CPUs emulation in future?

### DISCUSSION THREAD

**Discussion by macuserguru** [utmapp/UTM#5231] [General]
Hi,
did you change to QEMU 7.2.1 or to QEMU 8.0.0, but 8.0.0 don't support X86 32 Bit.

Read about in an German IT Magazine:
Schluss mit 32-Bit-x86-Hosts: QEMU 8.0.0 ist da
Mit Version 8.0.0 führt QEMU viele kleine, aber feine Änderungen ein. Vor allem aber streicht die freie Virtualisierungssoftware den 32-Bit-x86-Support. 
[https://www.heise.de/news/QEMU-streicht-den-Support-von-32-Bit-x86-Hosts-8975215.html?wt_mc=nl.red.ho.ho-nl-newsticker.2023-04-22.link.link](url)

Keine 32-Bit-Hosts mit x86 mehr
Herausragend ist hingegen vor allem, was es nicht mehr gibt. Beim Vorgänger, dem 7.x-Zweig, konnten Nutzer auch betagte x86-Systeme mit 32-Bit-CPUs einsetzen, um beispielsweise Retrosysteme von 68k über Alpha und MIPS bis hin zu SPARC zu emulieren. Das QEMU-Team hält nach eigener Aussage im Changelog "die Unterstützung von 32-Bit-x86-Hosts für die Systememulation nicht mehr für eine effektive Nutzung seiner begrenzten Ressourcen und beabsichtigt daher, diese einzustellen." Die begrenzten Ressourcen beziehen sich dabei offensichtlich nicht auf die 32-bittigen x86-Plattformen, sondern auf die Arbeitszeit der Entwickler. Wer also QEMU auf entsprechender Hardware nutzt, muss auf das Upgrade verzichten [und bei 7.x bleiben](https://www.heise.de/news/QEMU-7-0-mit-Intel-AMX-Support-und-viel-Neuem-fuer-ARM-RISC-V-7061188.html) – oder seine Projekte auf amd64-Hardware oder den Raspberry Pi migrieren.

nice greetings
Fritz (work on an M1 MacMini)

**Comment by osy** [comment]
We use 7.2.0 but will plan on updating to 8.0.0. Deprecated != removed. The feature is still there, it’s just not going to be improved.

**Reply by macuserguru** [reply]
When can you expect 8.0.0 in UTM?
I have UTM running from the App Store, but started with the free download.

**Reply by osy** [reply]
No plans anytime soon. There’s nothing in v8.0.0 that’s particularly interesting for us.
